provision.sh

```diff
@@ -44,20 +43,50 @@ echo "---------------------------------------------"
 # install nvm
 su - vagrant -c "wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash"
 
-# source bash
-su - vagrant -c "source ~/.bashrc"
+# source bash and install node
+su - vagrant <<NVM
 
-# install node
-su - vagrant -c "nvm install v4.4.3"
+source ~/.nvm/nvm.sh
+echo 'sourcing done'
+nvm install 4.4.3
+
+NVM
 
 echo "---------------------------------------------"
 echo "--- Installing Workshopper modules ----------"
 echo "---------------------------------------------"
-## install nodeschool-workshopper modules
 
-su - vagrant -c "npm i -g javascripting learnyounode git-it how-to-npm scope-chains-closures stream-adventure"
-su - vagrant -c "npm i -g elementary-electron how-to-markdown functional-javascript-workshop expressworks"
-su - vagrant -c "npm i -g promise-it-wont-hurt count-to-6"
+su - vagrant <<WORKSHOPPERS
+
+source ~/.nvm/nvm.sh
+echo "Installing javascripting ..."
+npm i -g javascripting
+echo "Installing learnyounode ..."
+npm i -g learnyounode
+echo "Installing git-it ..."
+npm i -g git-it
+echo "Installing how-to-npm ..."
+npm i -g how-to-npm
+echo "Installing scope-chains-closures ..."
+npm i -g scope-chains-closures
+echo "Installing stream-adventure ..."
+npm i -g stream-adventure
+echo "Installing elementary-electron ..."
```

Comment:
@alayek for using this, users can run the vagrant UI version and use it. Can't they?